lib/sm/client.rb

```diff
@@ -110,6 +111,8 @@ def connection
         conn.use :breakers
         conn.request :multipart
         conn.request :json
+        # conn.request :curl, ::Logger.new(STDOUT), :warn
```

Comment:
I don't think there's any problem with merging it, especially since the gem is only included in development, I just wonder if there's any point in doing so with it commented out. Is it just so that you can quickly uncomment it when you run into a problem to get some debug information? If that's the case, I think it's fine but might add a comment above the commented-out line that includes the middleware about what's up.